inf-ruby.el

```diff
@@ -709,7 +709,7 @@ Gemfile, it should use the `gemspec' instruction."
                 "bundle exec irb -I lib")
             "irb -I lib"))
          (name (inf-ruby-file-contents-match
-                gemspec "\\.name[ \t]*=[ \t]*\"\\([^\"]+\\)\"" 1))
+                gemspec "\\.name[ \t]*=[ \t]*[\'\"]\\([^\"\n]+\\)[\'\"]" 1))
```

Comment:
No need to escape the single quotes here. And why a newline inside the "not" brackets? I think a single quote should be there instead.